<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qnx::Internal::BlackBerryNDKSettingsWidget</class>
 <widget class="QWidget" name="Qnx::Internal::BlackBerryNDKSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="wizardLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Get started and configure your environment:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextBrowserInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wizardButton">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>13</red>
             <green>131</green>
             <blue>220</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>13</red>
             <green>131</green>
             <blue>220</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>13</red>
             <green>131</green>
             <blue>220</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>13</red>
             <green>131</green>
             <blue>220</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>144</red>
             <green>141</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>150</red>
             <green>147</green>
             <blue>145</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>environment setup wizard</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>276</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>BlackBerry NDK Path </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Utils::PathChooser" name="sdkPath" native="true"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTableView" name="ndkInfosTableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
